src/types.rs

```diff
@@ -1272,7 +1294,21 @@ impl Term {
             }
         }
     }
-    /// A human-readable serialization of the Term.
+    /// A human-readable serialization of the `Term`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use term_rewriting::{Signature, parse_term};
+    /// let mut sig = Signature::default();
+    ///
+    /// let term = parse_term(&mut sig, "A B(x_) CONS(SUCC(SUCC(ZERO)) CONS(SUCC(ZERO) CONS(ZERO NIL)))")
+    ///     .expect("parse of A B(x_) CONS(SUCC(SUCC(ZERO)) CONS(SUCC(ZERO) CONS(ZERO NIL)))");
+    ///
+    /// assert_eq!(term.display(&sig), ".(.(A B(x_)) CONS(SUCC(SUCC(ZERO)) CONS(SUCC(ZERO) CONS(ZERO NIL))))");
```

Comment:
I don't think we need the `term.display` assert here, since there's already one in `Term::display`. If you agree, please remove.